stylus/components/counter.styl

```diff
@@ -0,0 +1,20 @@
+@require '../settings/palette'
+@require '../tools/mixins'
+
+$counter
+    display inline-flex
+    align-items center
+    justify-content center
+    border-radius 50%
+    overflow hidden
+    background-color dodgerBlue
+    color white
+
+$counter--medium
+    width rem(40)
+    height rem(40)
+    font-size rem(20)
```

Comment:
It seems that for perfect alignment, a negative top margin should be applied.

![image](https://user-images.githubusercontent.com/465582/47356773-43d90e80-d6c5-11e8-94e1-3fcf2688a049.png) ![image](https://user-images.githubusercontent.com/465582/47356779-4b98b300-d6c5-11e8-98d9-d03ef6d48ee6.png)